README.md

```diff
@@ -150,6 +150,10 @@ properties:
 
 ## Dependency management
 To add a new dependency to the project, add the package information to glide.yaml and execute `glide up -v`
+* NOTE: Do not run the package manager version of glide. It may very well generate a broken vendor/ folder. Instead, use:
```

Comment:
"Do not run the package manager version of glide" becomes superstition very quickly - write something like "the version xyz of glide shipped with Fedora abc is known not to work properly"

Reply:
I get your point, but then being too precise might steer people in the wrong direction again. I haven't bisected all versions packaged for different versions of Fedora. In other words, I am not sure that the only version of glide affected is 0.13.1 for Fedora 29.

I updated the text in a way it would have been useful for me when I hit the problem earlier. Let me know how it works for you.